src/frontend/Komponenter/Personoversikt/Behandlingsoversikt.tsx

```diff
@@ -65,18 +68,19 @@ const Behandlingsoversikt: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
         });
     };
 
-    const startRevurdering = (fagsakId: string) => {
-        axiosRequest<Ressurs<void>, { fagsakId: string }>({
-            method: 'POST',
-            url: `/familie-ef-sak/api/revurdering/${fagsakId}`,
-            data: { fagsakId: fagsakId },
-        }).then((response) => {
-            if (response.status === RessursStatus.SUKSESS) {
-                hentFagsak();
-            } else {
-                settFeilmelding(response.frontendFeilmelding || response.melding);
-            }
+    const sorterBehandlingerNyesteFørst = (behandlinger: Behandling[]) => {
+        const listeKopi = [...behandlinger];
+        listeKopi.sort((a, b) => {
+            return compareDesc(new Date(a.opprettet), new Date(b.opprettet));
         });
+        return listeKopi;
```

Comment:
Litt usikker, men er det nødvendig å lage kopi her? `behandlinger.sort(....)` vil vel ikke mutere den opprinnelige listen siden det ligger in en egen funksjon? 🤔